Source/Tools/LinuxEmulation/VDSO_Emulation.cpp

```diff
@@ -553,6 +621,7 @@ void* LoadVDSOThunks(bool Is64Bit, FEX::HLE::SyscallHandler* const Handler) {
     VDSODefinitions[3].ThunkFunction = FEX::VDSO::x32::Handler_clock_gettime64;
     VDSODefinitions[4].ThunkFunction = FEX::VDSO::x32::Handler_clock_getres;
     VDSODefinitions[5].ThunkFunction = FEX::VDSO::x32::Handler_getcpu;
+    ///< VDSODefinitions[6] which is getrandom doesn't exist on 32-bit.
```

Comment:
Slightly misleading. This is clearer:

```suggestion
    ///< getrandom doesn't exist on 32-bit, so leave VDSODefinitions[6] unfilled
```